docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -216,15 +216,15 @@ Once the component has been released as unstable, you should perform additional
   - Use [queries](https://testing-library.com/docs/queries/about) like `getByRole` to find elements in the rendered DOM tree, and check that their attributes are as expected.
   - Avoid using snapshot tests.
   - When deciding what tests to write, generally the tests should cover all functionality in your component's state hook (`use{Component}`)
-  - Follow examples from other components for more guidance, such as [Button.test.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/src/components/Button/Button.test.tsx)
+  - Follow examples from other components for more guidance, such as [Button.test.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/library/src/components/Button/Button.test.tsx)
 - **Visual Regression Tests**
   - VR tests ensure that the components visuals do not change unexpectedly.
-  - Add stories to the `vr-tests-react-components` app, for example [Button.stories.tsx](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Button.stories.tsx)
+  - Add stories to the `vr-tests-react-components` app, for example [Button.stories.tsx](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx)
   - When deciding what tests to write, generally the tests should cover all states in your component's styling hook (`use{Component}Styles`).
 - **Bundle Size Tests**
-  - Example: [Button.fixture.js](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/bundle-size/Button.fixture.js)
+  - Example: [Button.fixture.js](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/library/bundle-size/Button.fixture.js)
 - **Manual Accessibility Tests**
-  - For more info see [Manual Accessibility Review Checklist](https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist)
+  - For more info see [Manual Accessibility Review Checklist](https://github.com/microsoft/fluentui/blob/3131d2710f6300504e096c794b707ea236ba4ce9/docs/react-v9/contributing/accessibility-review-checklist.md)
```

Comment:
lets use master branch refs instead commit that can get out of sync

```suggestion
  - For more info see [Manual Accessibility Review Checklist](https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/accessibility-review-checklist.md)
```

Reply:
ah, sure, did't notice, good catch!